DEPLOYMENT.md

```diff
@@ -1,10 +1,12 @@
-# [DRAFT] Deployment of workshoppers
+# Deployment of workshoppers
 
-Suggesting steps to organize the process of publishing changes from workshoppers to npm.
+Below you can find details regarding the deployment process of [workshoppers on npm](https://www.npmjs.com/org/workshopper). 
+This information is mainly aimed to be used by [workshopper teams](https://github.com/orgs/workshopper/teams).
 
-## Workshoppers versions
+## Workshoppers versioning
 
-Below, you can find a dashboard monitoring differences in versions of workshopper packages between github and npm.
+In the table below you will find an overview of package versioning
+between latest release in github and npm.
 
 workshopper | NPM version   | Master of workshopper | Build | NodeJs LTS compatible |
```

Comment:
"Master of workshopper" === "Master branch of github repository of workshopper"
I welcome ideas. I'm not sure how it relates to "NodeJs LTS compatible" to be honest.